spec/services/activitypub/fetch_remote_status_service_spec.rb

```diff
@@ -9,6 +9,9 @@
 
   let!(:sender) { Fabricate(:account, domain: 'foo.bar', uri: 'https://foo.bar') }
 
+  let(:follower) { Fabricate(:account, username: 'alice') }
+  let(:follow) { nil }
+  let(:response) { { body: Oj.dump(object), headers: { 'content-type': 'application/activity+json' } } }
```

Comment:
to be able to put the tests in the context where it made sense for them to be i had to hoist some params up to this top level and change how things got stubbed/called

- previously we always passed a prefetched body in rather than using the stub that is set up, so we never actually were testing the underlying fetch method. seems like there should be some way to parameterize these tests and run them once each if we wanted to cover both legs, being passed either as prefetched body or fetched via the stub, but idk how to do that in rspec
- allowed the whole response to be modified rather than just the object